src/LibraryManager/Providers/Cdnjs/CdnjsProvider.cs

```diff
@@ -237,35 +239,21 @@ public string GetSuggestedDestination(ILibrary library)
 
         private async Task<Stream> GetStreamAsync(ILibraryInstallationState state, string sourceFile, CancellationToken cancellationToken)
```

Comment:
>GetStreamAsync [](start = 35, length = 14)

We should make sure to be not on the UI thread here

Reply:
We are not on the main thread here.

---
In reply to: [197969429](https://github.com/aspnet/LibraryManager/pull/230#discussion_r197969429) [](ancestors = 197969429)